.github/workflows/release-enigma-go.yml

```diff
@@ -22,6 +22,8 @@ jobs:
           ref: master
           fetch-depth: 0
           token: ${{ secrets.GH_ACCESS_TOKEN}}
+      - name: Setup Git environment
+        uses: ./.github/actions/git-env-setup
```

Comment:
You have to add this YAML file under .github dir https://github.com/qlik-trial/qlik-cli/blob/master/.github/actions/git-env-setup/action.yml

Reply:
uses command is used to import Github action.

Reply:
Thanks! 🙇